src/maintainer/knowledge_base.rst

```diff
@@ -1028,10 +1028,137 @@ In order to qualify as a noarch python package, all of the following criteria mu
   which builds on Linux `and` Windows, with ``build_number`` offsets to create a pair packages, like
   ``dataclasses``.
 
+.. hint::
+  
+  You can build platform-specific ``noarch`` packages to include runtime requirements depending on the target OS.
+  See mini-tutorial below.
+
 If an existing python package qualifies to be converted to a noarch package, you can request the required changes
 by opening a new issue and including ``@conda-forge-admin, please add noarch: python``.
 
+.. _os_specific_noarch:
+
+Noarch packages with OS-specific dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to build ``noarch`` packages with runtime requirements that depend on the target OS (Linux, Windows,
+MacOS), regardless the architecture (amd64, ARM, PowerPC, etc). This approach relies on four concepts:
+
+1.  Virtual packages. Prefixed with a double underscore, they are used by conda to represent properties of the running system 
+    as constraints for the solver. We will use ``__linux``, ``__win`` or ``__osx``, which are only present when
+    the running platform is Linux, Windows, or MacOS, respectively. ``__unix`` is present in both Linux and MacOS. Note
```

Comment:
```suggestion
    the activated environment is Linux, Windows, or MacOS, respectively. ``__unix`` is present in both Linux and MacOS. Note
```

Reply:
I think one can only change the virtual packages with `CONDA_OVERRIDE_*` env variables. Otherwise, regardless the active environment, it will still report the native platform type.

Reply:
OK, thanks for the clarification. I don't know if "running platform" is defined somewhere?

Reply:
also, it is not clear to me from "as constraints for the solver" that this happens at runtime, and not while building the package.

Reply:
I'll rephrase, thanks!